app/config/services.yml

```diff
@@ -66,18 +66,18 @@ services:
         tags:
             - { name: security.voter }
 
-    # Uncomment the following lines to define a service for the Post Doctrine repository.
+    # Defines a service for the Tag Doctrine repository.
     # It's not mandatory to create these services, but if you use repositories a lot,
     # these services simplify your code:
     #
-    # app.post_repository:
-    #     class:     Doctrine\ORM\EntityRepository
-    #     factory:   ['@doctrine.orm.entity_manager', getRepository]
-    #     arguments: [AppBundle\Entity\Post]
+    app.tag_repository:
```

Comment:
thus, defining repositories as services encourages people to inject extra dependencies in it, without realizing that Doctrine will not use the container to get the repository, and so may give you the instance without the extra injections being done (which would be a total nightmare to debug, and very hard to avoid if you don't want to force instantiating all repositories in each request)